draft-ietf-acme-acme.md

```diff
@@ -2024,8 +2031,9 @@ retries, servers SHOULD rate-limit such requests.
 ## HTTP Challenge
 
 With HTTP validation, the client in an ACME transaction proves its control over
-a domain name by proving that for that domain name it can provision resources
-to be returned by an HTTP server. The ACME server challenges the client to
+a domain name by proving that can provision HTTP resources on a server with that
```

Comment:
typo: "proving that can provision HTTP resources" 

Maybe: "proving that the client can provision HTTP resources"?


"on a server with that domain name" seems a little loose. Maybe "On a server accessible under that domain name"?